File: MyFirstprogram/app/src/main/java/com/example/myfirstprogram/MainActivity.java
package com.example.myfirstprogram;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void my_function(View view) {
                        EditText name = findViewById(R.id.etName);


                    TextView output = findViewById(R.id.tvOutput);

                        String s = name.getText().toString();
                        output.setText( s );
    }
}

File: MyApplication2/app/src/main/java/com/example/myapplication/MainActivity.kt
package com.example.myapplication

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.EditText
import android.widget.TextView


class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)




    }



    fun convert_feet_to_inches(view: View) {

        var etval = findViewById<EditText>(R.id.etValue);
        var value_str = etval.text.toString() ;
        var value_int = Integer.parseInt( value_str );
        var inches = value_int * 12 ;

        var output = findViewById<TextView>(R.id.tvOutput);

        var msg = value_str + "Feet = " +  inches.toString() + "inches " ;

        output.setText( msg ) ;
    }
}







